web/static/js/components/idea_submission_form.jsx

```diff
@@ -66,41 +62,62 @@ export class IdeaSubmissionForm extends Component {
     this.setState({ category: event.target.value })
   }
 
+  handleAssigneeChange(event) {
+    this.setState({ assigneeId: Number.parseInt(event.target.value, 10) })
+  }
+
   render() {
-    const disabled = !this.state.body.length
+    const { users, stage } = this.props
+    const { assigneeId, body, ideaEntryStarted, category } = this.state
+    let disabled = !body.length
+    if (stage === ACTION_ITEMS) {
+      disabled = !(body.length && assigneeId)
+    }
+    const assigneeOptions = users.map(({ id, name }) =>
+      <option key={id} value={id}>{name}</option>
+    )
+    const defaultOption = (<option key={0} value={null}> -- </option>)
+
     const defaultCategoryOptions = [
       <option key="happy" value="happy">happy</option>,
       <option key="sad" value="sad">sad</option>,
       <option key="confused" value="confused">confused</option>,
     ]
+    const label = text =>
+      <div className={`${styles.pointingLabel} floating ui pointing below teal label`}>
+        {text}
+      </div>
     let pointingLabel = null
-
-    if (!this.state.ideaEntryStarted && this.props.stage === IDEA_GENERATION) {
-      pointingLabel = (
-        <div className={`${styles.pointingLabel} floating ui pointing below teal label`}>
-          Submit an idea!
-        </div>
-      )
+    let fieldArgs = []
+
+    if (!ideaEntryStarted && stage === IDEA_GENERATION) {
+      pointingLabel = label("Submit an idea!")
+      fieldArgs = ["category", category, this.handleCategoryChange, defaultCategoryOptions]
+    } else if (!ideaEntryStarted && stage === ACTION_ITEMS) {
+      pointingLabel = label("Create Action Items!")
+      fieldArgs = ["assignee", assigneeId, this.handleAssigneeChange, [defaultOption, ...assigneeOptions]]
     }
 
+    const field = (labelName, value, changeHandler, options) =>
+      <div className={`${styles.flex} five wide inline field`}>
+        <label htmlFor={labelName}>{`${capitalize(labelName)}:`}</label>
```

Comment:
hmmm... biggest thing here is that the `label` and `field` consts could be extracted out into its own component, like `SelectDropdown` (or something like that). That way, we keep `IdeaSubmissionForm` free of this extra markup and logic, and we get the added benefit of (I think) being able to reuse that component in the future.

Reply:
note: the latter suggestion is maybe premature to think about, but I do feel that extraction of the `label` and `field` components, here defined inline, should be extracted out into their own wrapping module.